From c6fd72aa73e9e2925aeb98781b35d00997c4d838 Mon Sep 17 00:00:00 2001
From: Koen Kooi <koen@dominion.thruhere.net>
Date: Sun, 2 Aug 2026 10:20:49 +0200
Subject: [PATCH] layer.conf: add wrynose to LAYERSERIES_COMPAT

No functional changes required for wrynose compatibility beyond what
walnascar/whinlatter already support -- this layer only touches
machine confs and a handful of recipes, none of which use anything
removed or renamed since walnascar. Needed to build any machine from
this layer (libretech-ac included) against a wrynose oe-core without
a hard "not compatible" fatal from bitbake-layers.

Upstream-Status: Pending

Signed-off-by: Koen Kooi <koen@dominion.thruhere.net>
---
 conf/layer.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/conf/layer.conf b/conf/layer.conf
index d40306c..1eac962 100644
--- a/conf/layer.conf
+++ b/conf/layer.conf
@@ -7,7 +7,7 @@ BBFILES += "${LAYERDIR}/recipes-*/*/*.bb ${LAYERDIR}/recipes-*/*/*.bbappend"
 BBFILE_COLLECTIONS += "meson"
 BBFILE_PATTERN_meson = "^${LAYERDIR}/"
 BBFILE_PRIORITY_meson = "1"
-LAYERSERIES_COMPAT_meson = "walnascar whinlatter"
+LAYERSERIES_COMPAT_meson = "walnascar whinlatter wrynose"
 
 # This should only be incremented on significant changes that will
 # cause compatibility issues with other layers
-- 
2.51.0

